### PR TITLE
[27607] Implement parent selection in wp breadcrumb, drop subject

### PR DESCRIPTION
**Pull request by oliverguenther**
- Adds inline-edit functionality to parent in WP view
- Removes the parent section from relations tab
- Changes relations count to no longer count parent

bildschirmfoto 2018-11-09 um 10 02 35
bildschirmfoto 2018-11-09 um 10 02 37

**Adds label with parent/hierarchy depending on ancestors**
bildschirmfoto 2018-11-09 um 10 03 14

**Adds inline-create when no parent is set**
bildschirmfoto 2018-11-09 um 10 03 22

https://community.openproject.com/wp/27607

### CODE REVIEW

File: frontend/src/app/components/wp-relations/wp-relations-create/wp-relations-autocomplete/wp-relations-autocomplete.upgraded.html [hunk 1]
```diff
@@ -1,8 +1,10 @@
 <form name="add_relation_form" class="form">
   <input type="text"
          class="wp-relations--autocomplete ui-autocomplete--input"
-         [ngClass]="{ '-error': noResults }"
-         [attr.placeholder]="text.placeholder">
+         [ngClass]="{ '-error': noResults && !initialSelection }"
+         (keydown.enter)="handleEnterPressed($event)"
+         (keydown.escape)="onEscapePressed.emit($event)"
```
Comment (HDinger) [inline frontend/src/app/components/wp-relations/wp-relations-create/wp-relations-autocomplete/wp-relations-autocomplete.upgraded.html:6]: On mobile this won't work, right? Maybe a click outside will close this, too.

**Review by HDinger — COMMENTED** [review]
There is still the issue when the parent title is very long. Then the icon is not shown and the parent not changeable. 
bildschirmfoto 2018-11-09 um 14 13 00

I played a little with the icons but did not found a better one. However I still think that especially deleting the parent is rather unintuitive. So maybe we could show a delete icon next to the edit icon?    I don't know if that is a good solution, but at least it is consistent to the WP page..

File: frontend/src/app/components/wp-relations/wp-relations-create/wp-relations-autocomplete/wp-relations-autocomplete.upgraded.html (hunk 1 above)
Comment (oliverguenther) [reply frontend/src/app/components/wp-relations/wp-relations-create/wp-relations-autocomplete/wp-relations-autocomplete.upgraded.html:6]: Good point, yes. I added a blur handler to do just that

**Comment by oliverguenther** [comment]
@HDinger I removed ellipsis for the parent element and fixed the blur issue. Good to go from my side.

**Pull request merged**